### PR TITLE
#161141274 check if user is verified

### PR DESCRIPTION
**Pull request by onwuvic**
#### What does this PR do?
It checks if a user is verified before logging in the user

#### Description of Task to be completed?
- create a condition that checks if the user has verified their account
- write a test to test the condition

#### How should this be manually tested?
- Sign up and try logging in without verifying your account it will throw an error
- Verify your account and try logging in, you will be logged in

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
Story type: feature
Story title: A user should verify their account before they can log in and perform any other authenticated operation.
Story ID: #161141274

#### Screenshots (if appropriate)
N/A
#### Questions:```
N/A

### CODE REVIEW

File: server/controllers/userController.js [hunk 1]
```diff
@@ -159,7 +159,14 @@ export default class UserController {
           return res.status(404)
             .send({ success: false, message: 'Invalid Email/Username or password' });
         }
-        const { id, email, username } = user.dataValues;
```
Comment (onwuvic) [inline server/controllers/userController.js:162]: Get users isVerified status from the returned user data

File: server/tests/user.test.js [hunk 2]
```diff
@@ -2,18 +2,28 @@
 import faker from 'faker';
 import chai, { expect } from 'chai';
 import chaiHttp from 'chai-http';
+import bcrypt from 'bcrypt';
 import db from '../models';
 import app from '../index';
 
 chai.use(chaiHttp);
 
+const hash = bcrypt.hashSync('password', 10);
```
Comment (onwuvic) [inline server/tests/user.test.js:11]: This hash the user password before saving it to the database

File: server/tests/user.test.js [hunk 3]
```diff
@@ -340,7 +371,7 @@ describe('Users', () => {
     it('Should return error message when user tries to login in with invalid password', (done) => {
       chai.request(app)
         .post('/api/v1/login')
-        .send({ emailOrUsername: 'testuser', password: 'wrongpassword' })
```
Comment (onwuvic) [inline server/tests/user.test.js:343]: I used a verified user here to test it.

**Review by oladimillion — COMMENTED** [review]
LGTM

**Review by s-akinrele — APPROVED** [review]

**Pull request merged**